### ISSUE TITLE
AUR installation fails due to no packages being found

### ISSUE THREAD

**Issue by flamusdiu** [aurapm/aura#458]
Just noticed this today. Apparently, `aura` isn't search AUR properly to install packages. 

```
~> sudo aura -Aakx google-chrome-beta --auradebug
User              => root
True User         => xxx
Build User        => xxx
Using Sudo?       => Yes!
Pacman Flags      => 
Other Flags       => AURInstall DelMDeps DiffPkgbuilds Unsuppress Debug
Other Input       => google-chrome-beta
Language          => English
Pacman Command    => pacman
Editor            => vi
$CARCH            => x86_64
Ignored Pkgs      => 
Build Path        => /home/pacman/pkg/
Pkg Cache Path    => /home/pacman/pkg/
Log File Path     => /var/log/pacman.log
Quiet?            => No.
Silent Building?  => No.
Must Confirm?     => Yes!
Needed only?      => No.
PKGBUILD editing? => No.
Diff PKGBUILDs?   => Yes!
Rebuild Devel?    => No.
Use Customizepkg? => No.
Forego PowerPill? => No.
Keep source?      => No.
aura >>= The following are not AUR packages:
google-chrome-beta

aura >>= No valid packages specified.
```
```
~> aura --version
                       
 __ _ _  _ _ _ __ _    Pacman v5.0.2 - libalpm v10.0.2
/ _` | || | '_/ _` |   Copyright (C) 2006-2016 Pacman Development Team
\__,_|\_,_|_| \__,_|   Copyright (C) 2002-2006 Judd Vinet
AURA Version 1.3.9     
 by Colin Woodbury     This program may be freely redistributed under
                       the terms of the GNU General Public License.
Aura Translators:      
 Chris "Kwpolska" Warrick (Polish)
 Denis Kasak / "stranac" (Croatian)
 Fredrik Haikarainen (Swedish)
 Lukas Niederbremer / Jonas Platte (German)
 Alejandro Gómez / Sergio Conde (Spanish)
 Henry "Ingvij" Kupty / Thiago "thiagowfx" Perrotta (Portuguese)
 Ma Jiehong / Fabien Dubosson (French)
 Kyrylo Silin (Russian)
 Bob Valantin (Italian)
 Filip Brcic (Serbian)
 "chinatsun" (Norwegian)
 "pak tua Greg" (Indonesia)
```
```
~> aura -Qi aura-bin
Name            : aura-bin
Version         : 1.3.9-1
Description     : A secure package manager for Arch Linux and the AUR written in
                  Haskell - Prebuilt binary
Architecture    : x86_64
URL             : https://github.com/fosskers/aura
Licenses        : GPL-3
Groups          : None
Provides        : aura
Depends On      : gmp  pacman  pcre
Optional Deps   : powerpill: For faster repository downloads.
                  customizepkg: For auto-editing of PKGBUILDs.
                  aur-git: AUR package completions for zsh. [installed]
Required By     : None
Optional For    : None
Conflicts With  : aura  aura-git
Replaces        : None
Installed Size  : 22.51 MiB
Packager        : Unknown Packager
Build Date      : Sun 11 Jun 2017 05:36:34 AM EDT
Install Date    : Sun 11 Jun 2017 05:36:43 AM EDT
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : None
```

**Comment by fosskers** [comment]
Hm, thanks for reporting this.

**Comment by flamusdiu** [comment]
Any luck with this? @fosskers

**Comment by theotherjimmy** [comment]
@flamusdiu I can't reproduce the problem with version 1.3.9.

**Comment by flamusdiu** [comment]
@theotherjimmy I'll check when I can. College killing me atm.

**Comment by theotherjimmy** [comment]
Thanks! Key me know wan m what you find, or of there are more steps to reproduce the problem.

**Comment by fosskers** [comment]
This seems solved on `master`, soon to be released as Aura 2.0.
